templates/worker_config.json.j2

```diff
@@ -0,0 +1,4 @@
+{\"auth_token\": \"{{ auth_token }}\",
```

Comment:
I am more confident about the two other file templates working than this one, due to the escape characters. I am not sure if they are meant to exist in the .json or if you added them because of the file creation through echo. At any rate: if you wish to merge these changes, test it all beforehand!